source/index.pug

```diff
@@ -0,0 +1,38 @@
+doctype html
+html
+  head
+    meta(charset='utf-8')
+
+    meta(name='viewport', content='width=device-width, initial-scale=1')
+    meta(name='description', content='Women and Web Literacy')
+
+    meta(property='og:type', content='website')
+    meta(property='og:site_name', content='Women and Web Literacy')
+    meta(property='og:locale', content='en_US')
+    meta(property='og:url', content='https://womenandweblit.mozilla.org')
```

Comment:
This will need to change to the Github pages url viz. `http://mozilla.github.io/womenandweb`